_data/communication.yml

```
@@ -99,6 +99,13 @@ websites:
       software: Yes
       doc: https://chatgrape.com/doc/deployment/2fa.html
 
+    - name: GroupMe
+      url: https://groupme.com/
+      img: groupme.png
+      tfa: Yes
+      sms: Yes
+      doc: https://help.groupme.com/hc/en-us/articles/360000579167-GroupMe-two-step-verification-FAQ
```

As it's a Zendesk page the article number is sufficient.
```suggestion
      doc: https://help.groupme.com/hc/en-us/articles/360000579167
```